### DISCUSSION TITLE
Help with UI issues

### DISCUSSION THREAD

**Discussion by rad-pat** [dbeaver/dbeaver#34694] [DBeaver Development] [answered]
I'm trying to pick up where Databend contribution left off. I have things building and running, but see 2 issues in the UI that I just need a pointer on how to resolve please. Firstly, sometimes I am getting large icons in the navigator instead of small icons, this seems to happen when the Databend connection is second in the list. If first then I get small icons. Secondly, the connection name label seems to overwrite the connection details in the tree, please see image below for reference:
<img width="440" alt="image" src="https://github.com/dbeaver/dbeaver/assets/45275736/33b27ded-f882-4531-a9df-c6ae7a2775d1">

**Comment by E1izabeth** [comment]
We don't experience this issue. I would like to help you, but I'm not even sure how to reproduce it.
Have you changed the source code? Maybe you could show me a branch?

**Comment by rad-pat** [comment, marked as answer]
I eventually managed to figure this out. The images provided in the Databend contribution were not the correct sizes. Now I have Databend png icons of the correct sizes - 16x16, 32x32, 64x64, 128x128 and the UI is now working correctly.
